src/Middleware/RequestBodyParserMiddleware.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace React\Http\Middleware;
+
+use Psr\Http\Message\ServerRequestInterface;
+use RingCentral\Psr7;
+
+final class RequestBodyParserMiddleware
+{
+    private $keepOriginalBody;
+    private $types = array();
+
+    /**
+     * @param bool $keepOriginalBody Keep the original body after parsing or not
+     */
+    public function __construct($keepOriginalBody = false)
```

Comment:
Shouldn't this be `true` by default to replicate PHP's default behaviour (and that of most PSR-7 implementations)?

Reply:
I agree :+1: I'd also vote to completely remove this parameter for now as the motivation is a bit unclear, plus we can still add this in a follow-up PR.

Reply:
Done :+1: